docs/scss/_community.scss

```diff
@@ -0,0 +1,32 @@
+.ov-community {
+    display:flex;
+    flex-direction: column;
+    > *:not(:last-child) {
+    margin-bottom: var(--ov--spacer--md);
+    }
+    }
+
+.ov-masthead--community {
+	    background: url('/images/oVirt_Community_Header.png') left top no-repeat var(--ov--color-brand--100);
+    .ov-heading--lg {
+        max-width: 400px;
+        margin-left: auto;
+        margin-right: auto;
+    } 
+   }
+   
+.ov-content-styles {
+	max-width: 1080px; 
+    list-style-type: disc;
```

Comment:
you'll want to apply this style to just `<ul>`s it won't have any affect on any other element.